### PR TITLE
anki.exporting.AnkiExporter.deckIds

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This change would help me simplify the code of
https://ankiweb.net/shared/info/1983204951

It would allow my exporter to use the code of AnkiExporter while
selecting which decks I export, instead of exporting all decks as I
was doing until today. (I never realized I was exporting all deck
actually, until someone sent me a bug report)

### CODE REVIEW

File: pylib/anki/exporting.py [hunk 1]
```diff
@@ -160,6 +160,12 @@ class AnkiExporter(Exporter):
     def __init__(self, col: _Collection) -> None:
         Exporter.__init__(self, col)
 
+    def deckIds(self) -> typing.List[int]:
```
Comment (dae) [inline pylib/anki/exporting.py]: The typing. prefix is unnecessary, but otherwise looks good.

Comment (Arthur-Milchior) [reply pylib/anki/exporting.py]: Removed.
Some Typing remains in the code; such as typing.Union. A reason to keep them at some places and not other ?

Comment (dae) [reply pylib/anki/exporting.py]: No reason to keep them - they were just added by a tool automatically, and nobody has gone through and manually removed the prefix and added the required import statement to the top.

**Comment by dae** [comment]
Thanks!

**Pull request merged**